docs/src/app/navigation.jsx

```diff
@@ -157,12 +171,139 @@ class CoreAppNavigation extends React.PureComponent {
         toggleNavigation();
     }
 
+    renderList() {
+        const {
+            classes,
+        } = this.props;
+        let sectionKeyNum = 0;
+        let sectionItemKeyNum = 0;
+        let subSectionItemKeyNum = 0;
+
+        return _.map(navigationItems, (section, index) => {
+            sectionKeyNum += 1;
+
+            return (
+                <React.Fragment key={`section-${sectionKeyNum}`}>
+                    {index > 0 && (
+                        <Divider
+                            className={classes.divider}
+                            color="inverse"
+                            inverse
+                            relaxed
+                        />
+                    )}
+
+                    <Header
+                        className={ClassNames(classes.header, classes.headerSection)}
+                        inverse
+                        size="medium"
+                        style={{ marginTop: 0 }}
+                    >
+                        {section.label}
+                    </Header>
+
+                    <ul
+                        className={classes.ul}
+                    >
+                        {_.map(section.items, (sectionItem) => {
+                            const path = sectionItem.path && `/${section.path}/${sectionItem.path}`;
+                            sectionItemKeyNum += 1;
+                            const key = `section--item-${sectionItemKeyNum}`;
+
+                            return this.renderListItem(
+                                sectionItem,
+                                path,
+                                key,
+                            );
+                        })}
+
+                        {_.map(section.subSections, (subSection) => {
+                            sectionItemKeyNum += 1;
```

Comment:
Doesn't at least part of you wanna rebel and do `sectionItemKeyNum++; // eslint-disable-line no-plusplus` ...? ;-)